extension.js

```diff
@@ -846,6 +855,12 @@ const ClipboardIndicator = Lang.Class({
 
     _toggleMenu: function(){
         this.menu.toggle();
+    },
+
+    _togglePrivatemode: function() {
+        this.privateModeMenuItem.toggle();
+        this._settings.set_boolean(Prefs.Fields.PRIVATEMODE, this.privateModeMenuItem.state);
+        this._onPrivateModeSwitch();
```

Comment:
I don't think this is necessary... shouldn't `toggle` trigger `_onPrivateModeSwitch`?

Reply:
no, as far as I understand, `toggle` only moves the UI element. Without calling `_onPrivateModeSwitch`, the UI will look like it's in private mode, but it will still remember clipboard snippets

Reply:
Are you sure? I just tried commenting out the two lower lines and things seemed to still work as expected.

Reply:
You are correct, those lines are redundant, not sure what I did wrong last time I checked. I removed them.

Reply:
Awesome, thanks! :)